web/business-registry-dashboard/nuxt.config.ts

```diff
@@ -67,6 +68,7 @@ export default defineNuxtConfig({
         'mdi:checkbox-multiple-blank-outline',
         'mdi:caret-down',
         'mdi:account-cog',
+        'mdi:information',
```

Comment:
Added icon here so it's preloaded. Otherwise, the library wants to fetch them from an external API which we don't have in our Content Security Policy list. I think it's better to do this than expand the CSP.

<img width="1448" height="282" alt="image" src="https://github.com/user-attachments/assets/05fbc5d1-d988-4004-a635-efc71aca19bb" />